### PR TITLE
feat: add compativility with olive realese

### PR DESCRIPTION
**Pull request by dcoa**
## Description

This PR adds support to Olive release updating the requirements and workflow dependencies.  Also, delete backends for old versions. 

## Testing instructions

- Install the plugin, you can use OPENEDX_EXTRA_PIP_REQUIREMENTS in config.yml file or [private plugins](https://docs.tutor.overhang.io/dev.html#xblock-and-edx-platform-plugin-development).
-  Follow the instructions [here](https://docs.google.com/document/d/18urdJPitAu1lUBvYdQcGXm4WRDbKD2eDf4TTGgrlzZU/edit#).

## Additional information
[JIRA ISSUE DS-366](https://edunext.atlassian.net/browse/DS-366)

## Checklist for Merge

- [ ] Tested in a remote environment
- [ ] Updated documentation
- [ ] Rebased master/main
- [ ] Squashed commits

### CODE REVIEW

File: .github/workflows/labeler.yml [hunk 1]
```diff
@@ -10,7 +10,7 @@ jobs:
     runs-on: ubuntu-latest
     name: Label the PR size
     steps:
-      - uses: CodelyTV/pr-size-labeler@v1.7.0
+      - uses: CodelyTV/pr-size-labeler@v1
```
Comment (JuanDavidBuitrago) [inline .github/workflows/labeler.yml:13]: There is an issue with this [version](https://github.com/CodelyTV/pr-size-labeler/issues/43), the size label doesn't add in PR. We have to continue using ``codelytv/pr-size-labeler@v1.7.0``version.

**Comment by JuanDavidBuitrago** [comment]
I tested this and it's working!

File: .github/workflows/labeler.yml (hunk 1 above)
Comment (dcoa) [reply .github/workflows/labeler.yml]: Thanks Juan, I've made the change.

**Comment by mariajgrimaldi** [comment]
This is what I get with the following configuration:
```
"THEME_OPTIONS": {
        "scripts": {
            "/login": [
                {
                    "content": "alert('This is a test for the inline script');",
                    "type": "inline"
                }
            ]
        },
        "theme": {
            "grandparent": "test-3",
            "name": "test-1",
            "parent": "test-2"
        }
    },
```
image
Is that OK? I see purple in the test file instead of red. Also, the scripts are not loading either. Not sure what I'm doing wrong (confession: I'm not that skilled with theming).

**Review by MaferMazu — APPROVED** [review]
I have the same color background as @mariajgrimaldi with her configuration, but in the header, I have things like TEST-1/logo-header
About the login script and the email test, they work ✨

**Comment by dcoa** [comment]
Do you run the comand `openedx-assets themes --theme-dirs /openedx/themes/ednx-test-themes/edx-platform --themes test-1`? 
If you did this then you override the default CSS files that contain the purple color (this is the default CSS folder https://github.com/eduNEXT/ednx-test-themes/tree/master/edx-platform/test-1/lms/static/css)

**Review by JuanDavidBuitrago — APPROVED** [review]

**Pull request merged**